packages/nix/polymc/default.nix

```diff
@@ -79,4 +79,17 @@ mkDerivation rec {
       --set GAME_LIBRARY_PATH ${gameLibraryPath} \
       --prefix PATH : ${lib.makeBinPath [ xorg.xrandr ]}
   '';
+
+  meta = with lib; {
+    homepage = "https://polymc.org/";
```

Comment:
Is this the URL we want right now?

Reply:
1.1.0 will be released after domain transfer anyways so let's keep that